src/views/cic/enter-date-birth.html

```diff
@@ -9,14 +9,13 @@
 
 
 {% block mainContent %}
-{% call hmpoForm(ctx, {attributes: {onsubmit: 'window.disableFormSubmit()'} }) %}
 
 {% set legendContent = translate("dateOfBirth.formLegend") %}
 {% set introductoryCopy = translate("dateOfBirth.dobInformationContext") %}
 {% set hintText = translate("dateOfBirth.hintText") %}
 {% set formInstructions = "<p class=\"govuk-hint\">"+hintText+"</p>" %}
 
-
+{% call hmpoForm(ctx, {attributes: {onsubmit: 'window.disableFormSubmit()'} }) %}
```

Comment:
What is this for? 👀

Reply:
Really good question, I was debugging and saw this was present in another file, added to test but forgot to remove - I believe the idea is to call `disableFormSubmit` which is defined globally on the window object when the hmpoForm macro is submitted, in order to prevent multiple submits - that being said I can't see the function being defined anywhere. 

Passport have defined their own function here https://github.com/govuk-one-login/ipv-cri-uk-passport-front/pull/181/files - not sure if this is something worth doing?

Reply:
In this case is there anything negative that could happen if a user clicks the button twice? The button action is to update the session model and navigate so I don't see too much issue with that

For things like post office search and submit on check details - this looks great! Let's get a tech debt ticket raised?